samples/subsys/video/capture/src/main.c

```diff
@@ -15,6 +16,48 @@ LOG_MODULE_REGISTER(main);
 
 #define VIDEO_DEV_SW "VIDEO_SW_GENERATOR"
 
+#define DISPLAY_FORMAT PIXEL_FORMAT_BGR_565
+
+#if DT_HAS_CHOSEN(zephyr_display)
+void display_setup(const struct device *const display_dev)
+{
+	struct display_capabilities capabilities;
+
+	if (!device_is_ready(display_dev)) {
+		LOG_ERR("Device %s not found", display_dev->name);
+		return;
+	}
+
+	printk("\nDisplay device: %s\n", display_dev->name);
+
+	display_get_capabilities(display_dev, &capabilities);
+
+	printk("- Capabilities:\n");
+	printk("  x_resolution = %u, y_resolution = %u, supported_pixel_formats = %u\n"
+	       "  current_pixel_format = %u, current_orientation = %u\n\n",
+	       capabilities.x_resolution, capabilities.y_resolution,
+	       capabilities.supported_pixel_formats, capabilities.current_pixel_format,
+	       capabilities.current_orientation);
+
+	display_set_pixel_format(display_dev, DISPLAY_FORMAT);
+}
+
+void video_display_frame(const struct device *const display_dev, struct video_buffer *vbuf,
+			 struct video_format fmt)
+{
+	struct display_buffer_descriptor buf_desc;
+
+	buf_desc.buf_size = vbuf->bytesused;
+	buf_desc.width = fmt.width;
+	buf_desc.pitch = buf_desc.width;
+	buf_desc.height = fmt.height;
+
+	display_write(display_dev, 0, 0, &buf_desc, vbuf->buffer);
+
+	display_blanking_off(display_dev);
```

Comment:
Why do we need to call `display_blanking_off` every time here? Can't we just call it once at init?

Reply:
Good remark. I will look at this. Thanks